package.json

```diff
@@ -44,22 +57,30 @@
     "@octokit/types": "13.5.0",
     "@octokit/webhooks": "13.2.8",
     "@octokit/webhooks-types": "7.5.1",
-    "@sinclair/typebox": "0.32.34",
+    "@sinclair/typebox": "0.32.35",
+    "@ubiquity-dao/ubiquibot-logger": "1.3.0",
     "dotenv": "16.4.5",
-    "jest": "29.7.0",
+    "hono": "4.4.13",
     "smee-client": "2.0.1",
+    "ts-node": "^10.9.2",
```

Comment:
Use `tsx`

Reply:
It is required: https://jestjs.io/docs/configuration